README.md

```diff
@@ -33,6 +33,8 @@ preparatory week. We hope you enjoy these days as much as we did preparing them!
     - [Installation For The Half-apple Cult](#macos)
 2. [The coolest team ever!](#contributors)
 
+Notes: insert mode: i;
+Undo: u 
 
```

Comment:
I believe these two lines are not supposed to be here. Delete them and you should be good to go